pkg/game/city/city_manager.go

```diff
@@ -121,13 +115,12 @@ func (manager *Manager) CanBePlaced(tile elements.PlacedTile, feat elements.Plac
 		// no City features or only has a completely new city
 		return true
 	}
-	citiesToJoin := manager.findCitiesToJoin(foundCities, tile)
+	citiesToJoin := manager.findCitiesToJoin(foundCities, feat.Sides)
 	// this may return an empty list for features that have no cities to join with
```

Comment:
nit: probably should move this comment
```suggestion
	// this may return an empty list for features that have no cities to join with
	citiesToJoin := manager.findCitiesToJoin(foundCities, feat.Sides)
```